README.md

```diff
@@ -22,7 +22,7 @@ To install the module dependencies before running the application, simply naviga
  Just run the Main.py. 
  You can understand the rest while using it for the first time.
  ### To execute
-    python Main.py
+    python News/Main.py
```

Comment:
Why this. ? Main.py is already in the root directory.

Reply:
Err... `Main.py` is not in root directory, its in `News` directory. If you run `python Main.py` from inside News directory, the config reader code will error out since its not path agnostic:

```
  File "{my path}/News-At-Command-Line/News/configReader.py", line 16, in __init__
    with open("config.yml", 'r') as ymlfile:
IOError: [Errno 2] No such file or directory: 'config.yml'
```

Does that make it clear?

Reply:
@adilansari  Ok. I thought it was in the root folder.